mlrun/datastore/targets.py

```diff
@@ -786,11 +786,17 @@ def add_writer_step(
         )
 
     def get_spark_options(self, key_column=None, timestamp_key=None):
-        return {
+        spark_options = {
             "path": store_path_to_spark(self._target_path),
             "format": "io.iguaz.v3io.spark.sql.kv",
-            "key": key_column,
         }
+        if isinstance(key_column, list) and len(key_column) >= 1:
+            spark_options["key"] = key_column[0]
+            if len(key_column) > 1:
+                spark_options["sorting-key"] = key_column[1]
```

Comment:
throw exception that spark engine doesn't support more than 2 entities

Reply:
i throw it in api.py. do you prefer it here?

Reply:
actually it's probably better here